mistralrs-server/src/chat_completion.rs

```diff
@@ -336,7 +336,22 @@ pub async fn chatcompletions(
             return ChatCompletionResponder::InternalError(e.into());
         }
     };
-    let sender = state.get_sender();
+    let sender = state.get_sender().unwrap();
+
+    // if the sender is closed, try to reboot the engine, and get the sender
+    // TODO: GS - make more readable?
```

Comment:
I assume you were trying to add your name here? If you could change this to `TODO(GS)` or something like that, it would make the code easier to grep and manage. Thanks!